src/current/v22.2/use-cloud-storage.md

```diff
@@ -253,9 +253,19 @@ The following S3 connection URI uses the `INTELLIGENT_TIERING` storage class:
 
 While Cockroach Labs supports configuring an AWS storage class, we only test against S3 Standard. We recommend implementing your own testing with other storage classes.
 
-{{site.data.alerts.callout_info}}
-[Incremental backups](take-full-and-incremental-backups.html#incremental-backups) are **not** compatible with the S3 Glacier Flexible Retrieval or Glacier Deep Archive storage classes. Incremental backups require ad-hoc reading of previous backups, which is not possible with the Glacier Flexible Retrieval or Glacier Deep Archive storage classes as they do not allow immediate access to S3 objects without first restoring the objects. See Amazon's documentation on [Restoring an archived object](https://docs.aws.amazon.com/AmazonS3/latest/userguide/restoring-objects.html) for more detail.
-{{site.data.alerts.end}}
+#### Incremental backups and archive storage classes
+
+[Incremental backups]({% link {{ page.version.version }}/take-full-and-incremental-backups.md %}#incremental-backups) are **not** compatible with the [S3 Glacier Flexible Retrieval or Glacier Deep Archive storage classes](https://docs.aws.amazon.com/AmazonS3/latest/userguide//storage-class-intro.html#sc-glacier). Incremental backups require ad-hoc reading of previous backups, which is not possible with backup files already in Glacier Flexible Retrieval or Glacier Deep Archive. This is because these storage classes do not allow immediate access to an S3 object without first restoring the object to its S3 bucket.
```

Comment:
Is this correct here? Although we say **not compatible**... It seems like a user can still use Glacier, but not when "actively" taking backups. Hopefully the few paragraph here capture that.